### ISSUE TITLE
English Subtitle 

### ISSUE THREAD

**Issue by fzhengnyc** [PopcornTimeTV/PopcornTimeTV#150]
Only English subtitle is not able to be loaded even if it is available.

**Comment by RefusedFlow** [comment]
I noticed the same issue @buzzel

**Comment by buzzel** [comment]
I'll have a look

**Comment by buzzel** [comment]
Is this issue still present? I have tried on several movies and shows and english subtitles do work for me. Sometimes the API bugs out and just fails. Unfortunately there isnt anything I can to do fix that if it just randomly loads or doesn't load subtitles. If it shows subtitles sporadically while the movie is playing. then we have an issue.

**Comment by fzhengnyc** [comment]
yes. it is still present. i am able to load other subtitles, but except English one.

**Comment by sasagr** [comment]
just tried the master today and I see the same issue.
Ex. Wayward Pines s01e08

**Comment by sasagr** [comment]
I tried the same episode a few minutes again but first I cleared the cache and this time it loaded the eng sub immediately. Actually the eng sub were already there even if the settings was off, but after setting it to eng they were still working. It seems that the cache, probably from a previous eng sub, was causing the issue. I m not sure if I will have time today to make other test but probably this is point to start looking at.

**Comment by Evhelden** [comment]
No issues here with English subtitles 

Regards,

Erwin van Helden
+31 6 46 99 88 93
Sent from iPhone

**Comment by fzhengnyc** [comment]
Thank you so much. I will try again and let you know.






On Thu, Jun 2, 2016 at 12:21 PM -0700, "Evhelden" <notifications@github.com> wrote:










No issues here with English subtitles 



Regards,



Erwin van Helden

+31 6 46 99 88 93

Sent from iPhone

**Comment by sasagr** [comment]
IT DOES NOT WORK and will explain. 
I did a fresh install of the 0.6.6 after cleaning up from any trace of old versions. 
I started with game of thrones latest episode and the eng sub was ok
Then I changed to outlander latest and game of thrones and also wayward pines latest two episodes and none of the was showing the eng sub tho it was present in the sub setting. The other language were working well. 
YOU CAN TRY IT YOURSELF
I believe it is something to do with the cache because the first episode I tried after the fresh install it was ok and also if I do remove the all cache it works (this I tried yesterday)
Hope somebody can make similar tests and find out what's going on

**Comment by RefusedFlow** [comment]
@sasagr Can you at least drop the caps dude! The API where subtitles are requested from isn't controlled by any of the team. I believe @buzzel was planning to implement a button to clear subtitle cache in the settings area in the future so hopefully this will resolve your problem when it's implemented.

Coming across as being angry and frustrated doesn't help the team in the slightest, just a little note for future issues you create.

**Comment by sasagr** [comment]
@RefusedFlow I did not mean to offend anybody so I edited my comment.
I m not frustrated or anything... I m just bringing my experience and giving some suggestion baed on my tests.
I just do not find useful answered from others who just say "no issue here..." while it is proved to be impossible that they don't have, since we are running the app in the same HW and SW environment.
That's all.
If I m mistaken and I can find the way to fix my isolated problem, that's good. If I can help to fix with my tests a common problem, that's even better
cheers and still congratulations to all of you who support such project
